src/govuk/components/notification-banner/_index.scss

```diff
@@ -43,6 +43,21 @@
       padding: govuk-spacing(4);
     }
 
+    // Wrap content at the same place that a 2/3 grid column ends, to maintain
+    // shorter line-lengths when the notification banner is full width
+    > * {
+      // When elements have their own padding (like lists), include the padding
+      // in the max-width calculation
+      box-sizing: border-box;
+
+      // Calculate the internal width of a two-thirds column...
+      $two-col-width: ($govuk-page-width * 2 / 3) - ($govuk-gutter * 1 / 3);
+
+      // ...and then take the border width and internal padding into account
+      $banner-exterior: (govuk-spacing(4) + $govuk-border-width);
```

Comment:
Should we save `govuk-spacing(4)` in a variable so we don't accidentally change the internal padding of the banner and forget to update the formula?

Reply:
Good idea 👍🏻  Done in 54ca621561401bd87ae7df07a2b4e371793c8cab.